beta/examples/tensorflow/classification/main.py

```diff
@@ -123,8 +123,8 @@ def resume_from_checkpoint(model, compression_ctrl, ckpt_path, steps_per_epoch,
     initial_step = model.optimizer.iterations.numpy()
     initial_epoch = initial_step // steps_per_epoch
 
-    scheduler_state = get_scheduler_state(initial_step, steps_per_epoch, config)
-    compression_ctrl.scheduler.load_state(scheduler_state)
+    controller_state = get_controller_state(initial_step, steps_per_epoch, config)
```

Comment:
just to clarify, this is the temporary solution that we were talking about, isn't it? in the future there should be a logic with getting controller state from the checkpoint.

Reply:
Yes, and this PR should make it easier to implement final logic